circleguard/gui.py

```diff
@@ -278,13 +278,6 @@ def __init__(self):
 
     def write(self, message):
         self.terminal.append(message)
-        self.scroll_to_bottom()
```

Comment:
lol thank god, I found this super annoying. Been meaning to make the change myself

Reply:
actually, I didn't test it, but does this keep you stuck to the bottom if you're already scrolled to the bottom? ie it should act as `scroll_to_bottom` if you're already there, and do nothing otherwise, just like twitch chat (for instance)

Reply:
Qt default behaviour should be the same as the twitch chat, but I will confirm this again to be sure

Reply:
confirmed to be the same behaviour as twitch without the hack